### PR TITLE
Improve "Testing" part of YourFirstPR.md

### PR DESCRIPTION
**Pull request by janpio**
### Motivation and Context
Testing instructions right now only prominently show running the whole test suite which takes quite some time which I noticed when replying to https://github.com/fastlane/fastlane/issues/11464#issuecomment-356604527

### Description
- Add instruction on how to run tests from one specific file
- Add subheadlines to Testing part for more structure
- Change code formatting so alternative testing commands are more prominent

### CODE REVIEW

**Review by KrauseFx — APPROVED** [review]
Nice 🎉

File: YourFirstPR.md [hunk 1]
```diff
@@ -42,13 +42,31 @@ You can also run those steps independently or on a more fine grained way.
...
+```
+bundle exec rspec [tool_name]
+```
+
+#### Unit tests in one specific test file
+
+If you know exactly which `_spec.rc` file you want to run, use
+
+```
+bundle exec rspec .\fastlane\spec\fastlane_require_spec.rb
```
Comment (revolter) [inline YourFirstPR.md]: Hmmm, Windows' back slashes, right?

**Comment by janpio** [comment]
In Windows both work, I think this happened via auto completion. Doesn't work for you?

File: YourFirstPR.md (hunk 1 above)
Comment (janpio) [reply YourFirstPR.md]: In Windows both work, I think this happened via auto completion.

Comment (revolter) [reply YourFirstPR.md]: No, on macOS and Linux, the separator is `/`, so I think it completely removes `\`:

> LoadError: cannot load such file -- /Users/iulian/Development/GitHub/fastlane/fastlane/.fastlanespecfastlane_require_spec.rb

**Pull request merged**